app/pages/onboarding/03-restore-wallet/restore-wallet.tsx

```diff
@@ -54,30 +57,32 @@ export const RestoreWallet: React.FC = () => {
   return (
     <Onboarding as="form" onSubmit={handleSecretKeyRestore}>
       <OnboardingTitle>Sign in to your wallet</OnboardingTitle>
-      <OnboardingText>
-        Sign in to your wallet by connecting your Ledger hardware wallet or a by entering your
-        Secret Key
-      </OnboardingText>
 
       {CONFIG.STX_NETWORK === 'mainnet' && (
         <>
+          <OnboardingText>
+            Connect your Ledger hardware wallet or enter your Secret Key
+          </OnboardingText>
           <OnboardingButton mt="extra-loose" onClick={() => history.push(routes.CONNECT_LEDGER)}>
             Continue with Ledger
           </OnboardingButton>
-
           <Hr mt="extra-loose" />
         </>
       )}
-
-      <Text textStyle="body.small.medium" mt="extra-loose" display="block">
-        Secret Key
-      </Text>
+      <Box>
+        <Text textStyle="body.small.medium" mt="extra-loose" display="block">
+          Secret Key
+        </Text>
+        <Text textStyle="caption" color="ink.600">
+          Your Secret Key is a series of random words also known as a seed phrase
```

Comment:
Let's add a "Learn more" link here at the end for https://www.hiro.so/questions/what-are-secret-keys-and-how-do-they-work